### ISSUE TITLE
Investigate the possibility to implement a disable attack system

### ISSUE THREAD

**Issue by godbit** [Lapsang-boys/Rex-Temporis#52]
By adding orbs to units you can choose which attack that orb enables. This means that if we have 0 attacks enabled on all of our towers but we add an orb enabling Attack 1 they should function as normal as long as we make sure that orb does not do anything but enable attack 1. My suggestion to use is lifesteal since that is almost 100% certain that we wont use anywhere else.

This then means that you can write a system which removes this orb from units at certain time -> disabling their attack for as long as the orb is gone.

**Comment by karlek** [comment]
Interesting! So this is how we would implement tower stunning functionality?

**Comment by godbit** [comment]
Yeah that could be one possible application! Another is to stop units from attacking whille "casting" a spell cast by a dummy or similar.

**Comment by karlek** [comment]
Ah, as you did with Cenarius. I like this, do we know if this works?

**Comment by godbit** [comment]
I did some research yesterday and i seems to work for other people. They have used it for systems that items enable attack 1 or 2. For instance a bow would enable attack 2 (ranged) and a sword would enable attack 1 (melee).

To easiest way to test this for us is probably just to implement it x) Maybe on a small scale and then create all the supporting methods later(?)
